src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/PseRespondToTribunalService.java

```diff
@@ -212,20 +217,30 @@ private String getHearingDate(CaseData caseData, SendNotificationType sendNotifi
      */
     public void sendClaimantEmail(CaseDetails caseDetails) {
         CaseData caseData = caseDetails.getCaseData();
+        boolean isWelsh = featureToggleService.isWelshEnabled()
+                && WELSH_LANGUAGE.equals(caseData.getClaimantHearingPreference().getContactLanguage());
+        String emailTemplate = isWelsh
+                ? cyNotificationToClaimantTemplateId
+                : notificationToClaimantTemplateId;
         if (YES.equals(caseData.getPseRespondentOrdReqCopyToOtherParty())) {
-            emailService.sendEmail(notificationToClaimantTemplateId,
+            emailService.sendEmail(emailTemplate,
                 caseData.getClaimantType().getClaimantEmailAddress(),
                 buildPersonalisationNotify(caseDetails));
         }
     }
 
     private Map<String, String> buildPersonalisationNotify(CaseDetails caseDetails) {
         CaseData caseData = caseDetails.getCaseData();
+        boolean isWelsh = featureToggleService.isWelshEnabled()
```

Comment:
Duplicated code. You already established the value of isWelsh, use it

Reply:
Moved into its own method